packages/browser-sdk/src/lib/react/Avatar/index.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from "react";
+
+import runes from "runes";
+
+interface AvatarProps {
+    avatarUrl?: string;
+    className?: string;
+    variant?: "outline" | "round" | "square";
+    name?: string;
+    size?: 32 | 36 | 40 | 60 | 80 | 200;
+}
+
+const getInitialsFromName = (name = "") => {
```

Comment:
Might be good to add some unit tests for this since the `name` is user input.

Reply:
done